src/ngraph/runtime/generic_cpu/gcpu_executable.hpp

```diff
@@ -437,6 +437,12 @@ class ngraph::runtime::gcpu::GCPUExecutable : public Executable
                                        broadcast_axes);
             break;
         }
+        case OP_TYPEID::BroadcastDistributed: {
+#ifdef NGRAPH_DISTRIBUTED_ENABLE
+            break; // It requires in-place output support
+#endif
+            break;
```

Comment:
We should throw an exception for unsupported ops, not silently ignore them.

Reply:
Added support for distributed broadcast op